src/server.rs

```diff
@@ -394,24 +389,25 @@ async fn index(
     let headers = merge_headers(&global_headers, req.headers()).await;
 
     // need a better name for this
-    let tuple_params =
-        match middleware_router.get_route(MiddlewareRoute::BeforeRequest, req.uri().path()) {
-            Some(((handler_function, number_of_params), route_params)) => {
-                let x = handle_http_middleware_request(
-                    handler_function,
-                    number_of_params,
-                    &headers,
-                    &mut payload,
-                    &req,
-                    route_params,
-                    queries.clone(),
-                )
-                .await;
-                debug!("Middleware contents {:?}", x);
-                x
-            }
-            None => HashMap::new(),
-        };
+    let tuple_params = match middleware_router
```

Comment:
Do you have a better name suggestion for `tuple_params`?

Reply:
I'm not sure about it, but as I think the purpose of the middleware is to modify the request before it is routed, I called it `modified_request`?

Reply:
`modified_request` is much better than `tuple_params` Thanks :D